packages/dds/tree/src/test/domains/json/twitter.ts

```diff
@@ -0,0 +1,631 @@
+    ) as Record<string, Record<string, number>>;
```

Comment:
I think I suggested using JSON.stringify() to serialize the state of your generator.  You don't need to leave it serialized in the source code though.

You can have a constant that points to the parsed object graph, like:

const twitterTextField = {
    "1": { ... and so on ...
 };